InnoTopicWebsite/src/app/skills/work-experience.component.html

```diff
@@ -31,80 +31,7 @@ <h1 class="section-title">Overview</h1>
 
 </section>
 
-<section id="languages" class="container">
-  <!-- FIXME: extract languages component -->
-  <h2 class="section-title">Languages</h2>
-  <!-- FIXME: 2..4 columns -->
-  <!--  <app-flags flags="de at lu pl gb us pt es es-ct fr" />-->
-  <!-- FIXME: extract component for flag + label-->
-  <div class="languages-grid">
-    <div class="language">
-      <div class="app-flags"><app-flags flags="pl" /></div>
-      <label>Polish</label>
-      <span>Native</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="gb us" /></div>
-      <label>English</label>
-      <span>Full professional proficiency</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="es mx" /></div>
-      <label>Spanish</label>
-      <span>Full professional proficiency</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="de at" /></div>
-      <label>German</label>
-      <span>Full professional proficiency</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="es-ct" /></div>
-      <label>Català</label>
-      <span>Intermediate</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="pt br" /></div>
-      <label>Portuguese</label>
-      <span>Intermediate</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="fr be" /></div>
-      <label>French</label>
-      <span>Beginner</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="it" /></div>
-      <label>Italian</label>
-      <span>Beginner</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="nl be" /></div>
-      <label>Dutch</label>
-      <span>Beginner</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="ru" /></div>
-      <label>Russian</label>
-      <span>Beginner</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="cn" /></div>
-      <label>Mandarin Chinese</label>
-      <span>Beginner</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="ma"/></div>
-      <label>Arabic</label>
-      <span>Beginner</span>
-    </div>
-    <div class="language">
-      <div class="app-flags"><app-flags flags="in"/></div>
-      <label>Hindi</label>
-      <span>Beginner</span>
-    </div>
-  </div>
-</section>
+<app-language-section />
```

Comment:
could we rename to app-natural-languages-section

natural languages, to distinguish from programming languages (overall convention i use in naming)